src/python/pants/releases/packages.py

```diff
@@ -241,22 +242,43 @@ def check_ownership(i, package):
     sys.exit(1)
 
 
-if sys.argv[1:] == ["list"]:
-  print('\n'.join(package.name for package in sorted(all_packages())))
-elif sys.argv[1:] == ["list", "--with-packages"]:
-  print('\n'.join('{} {} {}'.format(package.name, package.target, " ".join(package.bdist_wheel_flags)) for package in sorted(all_packages())))
-elif sys.argv[1:] == ["list-owners"]:
+def _create_parser():
+  parser = argparse.ArgumentParser()
+  subparsers = parser.add_subparsers(dest="command")
+  # list
+  parser_list = subparsers.add_parser('list')
+  parser_list.add_argument("--with-packages", action="store_true")
+  # list-owners
+  subparsers.add_parser("list-owners")
+  # check-ownership
+  subparsers.add_parser("check-ownership")
+  # build_and_print
+  parser_build_and_print = subparsers.add_parser("build_and_print")
+  parser_build_and_print.add_argument("version")
+  return parser
+
+
+args = _create_parser().parse_args()
+
+if args.command == "list":
+  if args.with_packages:
+    print('\n'.join(
+      '{} {} {}'.format(package.name, package.target, " ".join(package.bdist_wheel_flags))
+      for package in sorted(all_packages())))
+  else:
+    print('\n'.join(package.name for package in sorted(all_packages())))
+elif args.command == "list-owners":
   for package in sorted(all_packages()):
     if not package.exists():
-      print("The {} package is new!  There are no owners yet.".format(package.name), file=sys.stderr)
+      sys.stderr.write("The {} package is new! There are no owners yet.".format(package.name))
```

Comment:
Erm, clean except for this change. Missing a \n . Why this change?

Reply:
Hm was a kneejerk change from being burned with `print(b'', file=sys.stderr)` in https://github.com/pantsbuild/pants/pull/7073#issue-244303380 so not trusting `print(file=sys.stderr)` anymore.

It looks like it does work when using unicode though, so this change is not necessary. I'll revert that change.